kubernetes/helm/ckeditor-cs-stack/README.md

```diff
@@ -0,0 +1,51 @@
+# CKEditor Collaboration Server On-Premise stack Helm chart
+
+>:warning: **Not for production purposes!** This configuration is insecure, not persistent and its purpose is testing only.
+
+Helm chart for fast deployment of CKEditor Cloud Services in Kubernetes with MySQL database and redis cluster.
+
+## Getting started
+
+Download repository:
+```sh
+git clone git@github.com:cksource/ckeditor-cs-on-premises-infrastructure.git
+cd ckeditor-cs-on-premises-infrastructure/kubernetes/helm/ckeditor-cs-stack 
+```
+
+Create imagePullSecret for CKEditor container registry
+```sh
+# Your container registry username
+REGISTRY_USERNAME=
+# Your container registry authentication token
+REGISTRY_TOKEN=
+
+kubectl create secret docker-registry docker-cke-cs-com --docker-username $REGISTRY_USERNAME --docker-password $REGISTRY_TOKEN --docker-server https://docker.cke-cs.com
+```
+
+## Installation
+
+Update `values.yaml` file with your ingress hostname and licence key. Next install chart in your kubernetes cluster:
```

Comment:
> your ingress hostname

What does it mean? I have to change this:  
 

```
    hosts:
      - host: ckeditor-cs.organization.test
```

but for local devepolemnt what I should provide?